README.md

```diff
@@ -73,7 +73,7 @@ app.on("token", async ({ token, octokit }) => {
 
 require("http").createServer(getNodeMiddleware(app)).listen(3000);
 // can now receive user authorization callbacks at /api/github/oauth/callback
-```
+// See all endpoints at https://github.com/octokit/oauth-app.js#middlewares
```

Comment:
````suggestion
// See all endpoints at https://github.com/octokit/oauth-app.js#middlewares
```
````

Reply:
you accidentally removed the 3 backticks which end the code block syntax highlighting

Reply:
Oops! That is the last time I try to author through the web UI 🤦‍♂️